starter-apps/vue/vue-template/src/components/forms/Input.vue

```diff
@@ -0,0 +1,20 @@
+<script setup>
+const { id, name, label, placeholder, required } = defineProps([
+  'id',
+  'name',
+  'label',
+  'placeholder',
+  'required'
+])
```

Comment:
This seems to be missing some important props like `hint` and `error-message` for example.

Reply:
I've gotta be honest -- the biggest reason they're not here by default is because I only initially included required props. Are those props important enough that we need to make them required?